#############################################


#################################
bool elevator
int32 confidence
bool ultra_sound
bool ultra_sound_door
bool moving
bool laser
###############################

